ufersa/plataforma-sabia
# Add checkboxes to "be a reviewer" modal

## Summary


Resolves #576 

![Peek 2020-11-01 21-00](https://user-images.githubusercontent.com/10481028/97820252-11872200-1c8c-11eb-88ea-c33fb60ec8e2.gif)

## Relevant technical choices



## QA Steps


1. Login as a user that can request to be a reviewer
2. Go to the user's profile
3. Click "be a reviewer" button
4. Verify that you can submit the form only if there is at least one category + subcategory selected and all checkboxes are checked

## Checklist
